stablehlo/reference/Element.h

```diff
@@ -41,8 +44,60 @@ class Element {
       : type_(type), value_(std::make_pair(value.real(), value.imag())) {}
 
   Element(const Element &other) = default;
+  Element() = default;
   /// @}
 
+  /// @}
+  /// \name Value Generators
+  /// @{
```

Comment:
What are the benefits of having this functionality as methods vs constructors? Given that we already have constructors which take Type and something else, why not just add more constructors?